apps/docs/app/layout.tsx

```diff
@@ -77,7 +78,14 @@ export const metadata: Metadata = {
   },
 };
 
+// Cookie name for analytics concent
+const CONCENT_COOKIE_NAME = "rccn";
```

Comment:
- Change variable name to CONSENT_COOKIE_NAME
- Correct the consent spelling in comment